soroban-sdk-macros/src/lib.rs

```diff
@@ -353,14 +353,9 @@ struct ContractImportArgs {
 /// #[test]
 /// fn test() {
 ///     let env = Env::default();
-///
-///     // Define IDs for contract A and B.
-///     let contract_a_id = BytesN::from_array(&env, &[0; 32]);
-///     let contract_b_id = BytesN::from_array(&env, &[1; 32]);
-///
 ///     // Register contract A using the imported WASM.
-///     env.register_contract_wasm(&contract_a_id, contract_a::WASM);
-///
+///     let contract_a_id = env.register_contract_wasm(&contract_a_id, contract_a::WASM);
```

Comment:
`&contract_a_id` is still a parameter to the function. That looks like it is in error.

Reply:
Fixed.